code/game/objects/structures/kitchen_spike.dm

```diff
@@ -242,4 +242,96 @@
 			untie_mob(L)
 	return ..()
 
+// Non-Legion Flavour execution.
+
+/obj/structure/kitchenspike/gallow
```

Comment:
No no no no no no no.
Neither crosses nor gallows should be *kitchen spikes.*

Reply:
Looks like it's time for a refactor.